presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/LogsHandler.java

```diff
@@ -245,4 +293,28 @@ else if (nullEnd) {
             return s -> level.name().equals(s);
         }
     }
+
```

Comment:
If you mean the various `internalError` and similar functions, I'm not sure that's a great idea; combining the logging and responses should be done carefully to make sure it is appropriate for the situation at hand.

Though I think I _will_ make a utility class for common responses.